src/components/backstop/BackstopQueueMod.tsx

```diff
@@ -55,16 +55,18 @@ export const BackstopQueueMod: React.FC<PoolComponentProps> = ({ poolId }) => {
             poolId={poolId}
             q4w={{ exp: BigInt(0), amount: poolBackstopUserData.unlockedQ4W }}
             inTokens={poolBackstopUserEst.totalUnlockedQ4W}
+            first={true}
           />
         )}
         {poolBackstopUserData.q4w
           .sort((a, b) => Number(a.exp) - Number(b.exp))
-          .map((q4w) => (
+          .map((q4w, index) => (
             <BackstopQueueItem
               key={Number(q4w.exp)}
               poolId={poolId}
               q4w={q4w}
               inTokens={toTokens(q4w.amount)}
+              first={poolBackstopUserData.unlockedQ4W == BigInt(0) && index == 0}
```

Comment:
Why is this needed?

Reply:
Technically anything that is unlocked is still on the Q4W queue. In the UI, we separate these out into "unlocked shares" and a Q4W array of locked entries.

Thus, we only mark the first Q4W entry as the first (so the button is not disabled) if it is the first AND there is no unlocked entries before it.